src/logic/calculate.js

```diff
@@ -0,0 +1,33 @@
+const calculate = (dataObjt, buttonName) => {
+  let { total = '', next = '', operation = '' } = dataObjt;
+  switch (buttonName) {
+    case 'AC':
+      total = '';
+      next = '';
+      operation = '';
```

Comment:
- [ ] Since operate is going to be used here depending on the user's input, we need to import the operate function here.